### ISSUE TITLE
How do I rate levels in version 1.8?

### ISSUE THREAD

**Issue by Touchcreator** [Cvolton/GMDprivateServer#1129]
So I made a GDPS with version 1.8 but I'm confused because I don't know how I'm supposed to rate levels.

**Comment by Lila2048** [comment]
Just use !rate in the comments. IsAdmin in the accounts table needs to be true for this ti work.

**Comment by MegaSa1nt** [comment]
You forgot that this is 1.8, no accounts

**Comment by Lila2048** [comment]
Yeah true. I am unsure of what to do. The users table is where data is stored for unregistered (green) users. Maybe theres something in there? If you had to you can change the data in PhpMyAdmin if you cant find another way. I run a 2.2 gdps so I am not the best help in the world.
